### PR TITLE
Quick format for search-interview.html

### PR DESCRIPTION
**Pull request by apluscs**

### CODE REVIEW

**Review by GrantComm — APPROVED** [review]
LGTM

File: src/main/webapp/search-interview.html [hunk 1]
```diff
@@ -131,7 +131,7 @@ <h5 class="text-center lead">Select the time you want from the dropdown on the d
             <div class="modal-header">
               <h5 class="modal-title" id="modal-title"></h5>
               <button type="button" class="close" data-dismiss="modal">
-                <span aria-hidden="true">&times;</span>
+              <span aria-hidden="true">&times;</span>
```
Comment (meagancheese) [inline src/main/webapp/search-interview.html]: Shouldn't this one still be indented?

File: src/main/webapp/search-interview.html [hunk 2]
```diff
@@ -23,7 +23,7 @@
         <button class="navbar-toggler" type="button" data-toggle="collapse" 
           data-target="#navbarNavAltMarkup" aria-controls="navbarNavAltMarkup" 
           aria-expanded="false" aria-label="Toggle navigation">
-          <span class="navbar-toggler-icon"></span>
+        <span class="navbar-toggler-icon"></span>
```
Comment (meagancheese) [inline src/main/webapp/search-interview.html]: Shouldn't this one still be indented since it's inside the button tag?

**Review by meagancheese — CHANGES_REQUESTED** [review]

File: src/main/webapp/search-interview.html (hunk 2 above)
Comment (apluscs) [reply src/main/webapp/search-interview.html]: Yep, so hard to find a good formatter :/

**Review by meagancheese — APPROVED** [review]
LGTM

**Review by jalexanderqed — APPROVED** [review]

**Pull request merged**